c_api/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "cpp_demangle_c_api"
+version = "0.1.0"
+authors = ["Miguel Saldivar <saldivarcher@gmail.com>"]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib"]
+name = "cpp_demangle_c_api"
+path = "lib.rs"
```

Comment:
FWIW, I'd just use the conventional paths here (`src/lib.rs`) and not explicitly override them.

Reply:
Changed the directory structure, let me know if it's okay now :)